classes/migration/upgrade/v3_5_0/I10157_CreateAndMigrateIssueIdField.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace APP\migration\upgrade\v3_5_0;
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+
+class I10157_CreateAndMigrateIssueIdField extends \PKP\migration\Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
```

Comment:
This adds the column, but does not add the FK constraint that would be created on a fresh install.